lib/fs.js

```diff
@@ -1300,12 +1296,7 @@ StatWatcher.prototype.stop = function() {
 };
 
 
-var statWatchers = {};
-function inStatWatchers(filename) {
-  return Object.prototype.hasOwnProperty.call(statWatchers, filename) &&
-      statWatchers[filename];
-}
-
+const statWatchers = Object.create(null);
```

Comment:
@chrisdickinson  We store `StatWatcher` objects against file names. So you meant, a `Map`, right?